src/components/dropdown/dropdown.js

```diff
@@ -678,13 +677,15 @@ const Dropdown = Input(InputIcon(InputLabel(InputValidation(class Dropdown exten
       content.push(this.inputIconHTML('dropdown'));
     }
 
-    content.push(
-      <Portal onReposition={ this.calculatePosition }>
-        <div { ...this.listBlockProps } ref={ (node) => { this.listBlock = node; } }>
-          { this.listHTML }
-        </div>
-      </Portal>
-    );
+    if (this.state.open) {
```

Comment:
unfortunately we cannot do this due to https://jira-sage.valiantyscloud.net/browse/CARBON-1044

Reply:
This branch was approved tested and merged. Should we revert. This being said, when I read the other ticket, it seems that CARBON-1044 is resolved by this fix right?

Reply:
The issue is that automation expects the dropdown list to be present in the DOM, even when closed. Now that you have removed the list from the DOM, automation will fail.

The ticket I linked to needs a solution that will resolve the issue with automation as well.